client/src/components/NewVersionAvailable.css

```diff
@@ -0,0 +1,3 @@
+.reloadPageLink {
```

Comment:
I'd add a style rule to even out the whitespace in the dialog. Either remove the top margin on the p-tag, or inherit the bottom margin, as long as it's balanced.